commands/eval.md

```diff
@@ -8,9 +8,11 @@ These names of input keys are available to the script as the [_KEYS_ global runt
 Any additional input arguments **should not** represent names of keys.
 
 **Important:**
-to ensure the correct execution of scripts, both in standalone and clustered deployments, all names of keys that a script accesses must be explicitly provided as input key arguments.
+To ensure the correct execution of scripts, both in standalone and clustered deployments, all names of keys that a script accesses must be explicitly provided as input key arguments.
 The script **should only** access keys whose names are given as input arguments.
 Scripts **should never** access keys with programmatically-generated names or based on the contents of data structures stored in the database.
+If a key with expiry (TTL) exists at the start of eval, it will not get expired during the evaluation of a script.
+During the evaluation of a Lua script, TTL of a key can go down to zero. After that, it remains zero until the script terminates.
```

Comment:
Are you sure? Can you give an example?

I thought that we freeze time during evaluation, so TTL will return the same value every time throughout the evaluation.

Reply:
See redis/redis#10300

CC @enjoy-binbin

Reply:
yean, we will freeze the time during the evaluation, the [P]TTL should return the same value in this case in Redis 7.2